### ISSUE TITLE
Liberate Appraisal Data

### ISSUE THREAD

**Issue by spatialaustin** [open-austin/project-ideas#50]
#### Concise description. What problem are you trying to solve? 

Project with the goal of liberating Texas appraisal district data by making it avaialble in an open, usable format.

#### Link (more details/brain dump/alpha)

[open-austin/liberate-appraisal-data](https://github.com/open-austin/liberate-appraisal-data)

#### Who will use/benefit from this project?

Homeowners, renters, civic hackers, journalists, affordable housing advocates, everyone.

#### Project Needs (dev/design/resources)

Share what your project needs are here.

#### Status (experiment, alpha, beta, official)

Alpha

**Comment by McCnnll** [comment]
![liberate_appraisal_data](https://cloud.githubusercontent.com/assets/7535602/13550444/622c7d96-e2e3-11e5-843d-7a08d4dc121b.png)

**Comment by spatialaustin** [comment]
suggested name: Appraisal Liberation Movement

**Comment by decibel** [comment]
I'm available as a data architect.

**Comment by mateoclarke** [comment]
Might be worth writing a one-pager for people to understand the policy issue. Then distribute to the County Commissioners. Or even some candidates like Jeff Travillion and David Holmes.

**Comment by mateoclarke** [comment]
This could be an effective tactic:
http://chihacknight.org/blog/2016/02/16/chi-hack-nights-open-data-pledge-for-cook-county-clerk-of-the-circuit-court.html

**Comment by davemcphee** [comment]
I don't know exactly what the appraisal data is,  is this something that's in the travis tcad db?

**Comment by AustinBulldog** [comment]
I have published several in-depth investigations regarding the Travis Central Appraisal District (TCAD) and exposed numerous defects in the system. Here are links:
1. Appraisal records hidden from public view (Nov. 18, 2011). This piece resulted in immediate action by TCAD to make property owners prove they were entitled to confidentiality: http://www.theaustinbulldog.org/index.php?option=com_content&view=article&id=162:appraisal-records-hidden-from-public-view&catid=3:main-articles
2. Homestead exemptions rife with abuse (Dec. 20, 2013, Part 1 of 2): Hundreds of homeowners were not billed for all property taxes due because they had more than the one legal exemption: http://www.theaustinbulldog.org/index.php?option=com_content&view=article&id=266:homestead-exemptions-rife-with-abuse&catid=3:main-articles
3. Homestead exemptions a tax loophole (Feb. 27, 2014, Part 2 of 2): The appraisal district relied mostly on homeowners' statements and provided no scrutiny. At that point these articles had caused collection of more than $300,000 in back taxes because of these reports, which also exposed major problems in how data is kept and needs for improvements. http://www.theaustinbulldog.org/index.php?option=com_content&view=article&id=268:homestead-exemptions-a-tax-loophole&catid=3:main-articles

I would really like to be part of any efforts to help Liberate Appraisal Data.

**Comment by spatialaustin** [comment]
@davemcphee yes, it's the full TCAD database we're interested in. What year(s) do you have and can you give them to us?

**Comment by AustinBulldog** [comment]
I don’t have the full year. My appraisal data project only pulled the properties that had homestead exemptions. Now of those all I seem to have are spreadsheets. 

Ken

Ken Martin
Founder, Editor & Publisher
The Austin Bulldog



Nonprofit investigative Reporting in the Public Interest
Phone O: 512-474-1022 
e-mail ken@theaustinbulldog.org <mailto:ken@theaustinbulldog.org>
web www.theaustinbulldog.org <http://www.theaustinbulldog.org/>
Facebook: http://on.fb.me/i8sFU3 <http://on.fb.me/i8sFU3>
http://twitter.com/AustinBulldog <http://twitter.com/AustinBulldog>
http://www.linkedin.com/in/kenmartinaustin <http://www.linkedin.com/in/kenmartinaustin>
P.O. Box 4400 Austin TX 78765

**Comment by davemcphee** [comment]
Hi John,

I have the full 2015 dump, in mysqldump format, a little over 8GB. No idea
how to get it you :( 2 DVDs in the post? I know it's 2016 but 8GB is still
a decent file size...

**Comment by luqmaan** [comment]
Can you upload it to S3? https://aws.amazon.com/blogs/aws/amazon-s3-multipart-upload/

If you're willing to do that we can figure out the details on Slack.

**Comment by johnclary** [comment]
@AustinBulldog no worries. thanks for your interest. we will be drafting a letter to send to TCAD executives and would welcome your support.

**Comment by tylermarkham** [comment]
Hi all, just checking in on this. I'm thinking about buying the 2016 TCAD data but don't want to do so if anyone else already has.

**Comment by werdnanoslen** [comment]
@tylermarkham did you buy the data?

**Comment by tylermarkham** [comment]
Not quite yet. 

Tyler

**Comment by tylermarkham** [comment]
I am planning to buy the Tax Office (not appraisal office) data today though.
